terminus-ui/toggle/src/toggle.component.ts

```diff
@@ -104,7 +104,18 @@ export class TsToggleComponent extends TsReactiveFormBaseComponent {
    * Emit an event each time the toggle value changes
    */
   // TODO: Rename to avoid conflict with native events: https://github.com/GetTerminus/terminus-ui/issues/1472
+  /**
+   * @deprecated change will be deprecated, use 'selectionChange' instead
+   */
   // tslint:disable-next-line: no-output-native
   @Output()
   public readonly change: EventEmitter<TsSlideToggleChange> = new EventEmitter();
+
+  /**
+   * Emit an event each time the toggle value changes
+   */
+  // tslint:disable-next-line: no-output-native
```

Comment:
We don't need this line. It was to disable linting for `change` as it was rightly calling us out for using a native event name.